src/pde_solve_ns.jl

```diff
@@ -0,0 +1,84 @@
+struct NNPDENS{C1,C2,O,sde_algorithm} <: NeuralNetDiffEqAlgorithm
+    u0::C1
+    σᵀ∇u::C2
+    opt::O
+    sde_algorithm::sde_algorithm
+end
+
+NNPDENS(u0,σᵀ∇u;opt=Flux.ADAM(0.1),sde_algorithm=EM()) = NNPDENS(u0,σᵀ∇u,opt,sde_algorithm)
+
+function DiffEqBase.solve(
+    prob::TerminalPDEProblem,
+    alg::NNPDENS;
+    timeseries_errors = true,
+    save_everystep=true,
+    adaptive=false,
+    abstol = 1f-6,
+    reltol = 1f-5,
+    verbose = false,
+    maxiters = 300,
+    dt,
+    trajectories)
+
+    X0 = prob.X0
+    tspan = prob.tspan
+    ts = prob.tspan[1]:dt:prob.tspan[2]
+    d  = length(X0)
+    g,f,μ,σ,p = prob.g,prob.f,prob.μ,prob.σ,prob.p
+
+    data = Iterators.repeated((), maxiters)
+
+
+    #hidden layer
+    opt = alg.opt
+    u0 = alg.u0
+    σᵀ∇u = alg.σᵀ∇u
+    sde_algorithm = alg.sde_algorithm
+    ps = Flux.params(u0, σᵀ∇u...)
+
+    function F(h, p, t)
+        u =  h[end]
+        X =  h[1:end-1].data
```

Comment:
why are you taking the data part here?

Reply:
it is ```ERROR: LoadError: MethodError: no method matching Float32(::Tracker.TrackedReal{Float32})``` with case where σ or μ dependent from Tracked X

Reply:
Interesting that it still works with this... but since it works it seems to be fine.